resources/measures/HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -583,7 +583,7 @@ def self.process_infiltration_for_unit(model, runner, obj_name_infil, infil, win
       # Wind Driven Air Infiltration Calculations" by Walker and Wilson (1998)
 
       # Pressure Exponent
-      n_i = 0.67
+      n_i = 0.65
```

Comment:
Should this be stored in `constants.rb` if it's used in more than one place?

Reply:
It's only used in two places and this code is going to be replaced. Didn't seem worth it.